EIPS/eip-1344.md

```diff
@@ -5,7 +5,7 @@ author: Richard Meissner (@rmeissner), Bryant Eisenbach (@fubuloubu)
 discussions-to: https://ethereum-magicians.org/t/add-chain-id-opcode-for-replay-protection-when-handling-signed-messages-in-contracts/1131
 category: Core
 type: Standards Track
-status: Last Call
+status: Accepted
 created: 2018-08-22
 review-period-end: 2019-04-30
```

Comment:
Should remove this too.

However, core EIPs I do not think follow this logic. Accepted means that it was accepted into a hard fork.

Reply:
I thought this was what the `Final` status meant?

Reply:
`Final` for core EIPs currently means they went live in a hard fork.

Reply:
Should this be reverted then until it has been accepted?

Reply:
From EIP-1: "The process for Core Devs to decide whether to encode an EIP into their clients as part of a hard fork is not part of the EIP process. If such a decision is made, the EIP wil move to final."

Would seem to suggest that `Accepted` doesn't mean accepted into a hard fork for a Core EIP like this one.

Reply:
Please have a look at EIP-233 as well, that tries to capture the process, how it worked for past hard forks.

Reply:
EIP-233 stipulates all related EIPs must be moved to `Accepted` before the hard fork EIP is moved to `Accepted`, I don't think it's clear the inverse is true (nor do I think it should be).

Marking this as resolved for now, there doesn't seem to be a process issue with this change in  status.

Reply:
I think the problem here is that EIP-1 is very inconsistent as it stands today. Re-reading the status section to me it seems that once an EIP is in `Accepted` state it cannot be changed anymore (there is no way to move it to any other state).

The current version EIP-233 doesn't explain it well, but in the last hard fork the process was that everything (the meta as well as the referenced EIPs) turned `Accepted` when no more changes were proposed and everything turned `Final` when the hardfork went live.

Reply:
Hmm, perhaps it might make sense to add an "escape value" from `Accepted` to `Deferred` if the EIP is decided to not be implemented in any upcoming hard fork. That's probably already the consideration, to be honest.

Reply:
Can you draft up a fix to EIP-1 to make this clear?

Reply:
Done #2080

Reply:
I think #2196 solves this discussion.